server/network/aoprotocol.py

```diff
@@ -430,8 +430,10 @@ def net_cmd_ms(self, args):
             return
         if sfx_delay < 0:
             return
-        if button not in (0, 1, 2, 3, 4):
-            return
+        if '4' in button and "<and>" not in button:
+            if not button.isdigit():
```

Comment:
Clause can be folded into above if-statement.

Reply:
Fair enough